src/routes/tools/extreme-weather/index.json.js

```diff
@@ -0,0 +1,22 @@
+import { getToolContent } from "../_common/server-utils";
+
+export function get(_req, res) {
+  try {
+    const {
+      description,
+    } = require("/content/tools/extreme-weather/description");
```

Comment:
Note: for content that is better suited to live in `.js` or `.json` files we can just use `require` instead of `fs.readFileSync`. Although I should probably update the `getToolContent` function to handle that...

Reply:
It appears that NodeJS / Webpack / Sapper won't let you use `require` from an imported function so leaving as is.